examples/src/app/index.tsx

```diff
@@ -96,8 +96,8 @@ const MainLayout = () => {
                             const e = new p.example();
                             const assetsLoader = e.load;
                             const controls = e.controls;
-                            return <Route key={`/iframe${p.path}`} path={[`/iframe${p.path}`, `${p.path}.html`]}>
-                                <ExampleIframe controls={controls} assets={assetsLoader ? assetsLoader().props.children : null} files={p.files}/>
+                            return <Route key={`/iframe${p.path}`} path={[`/iframe${p.path}`]}>
+                                <ExampleIframe controls={controls} assets={assetsLoader ? assetsLoader().props.children : null} files={p.files} debug={p.path.includes('mini-stats')} />
```

Comment:
I hoped this could be less hardcoded .. maybe specified by a flag inside the example? Or at least have a list of examples that use debug version inside index.tsx or similar.